Description: Upstream changes introduced in version 3.9.0-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 zorp (3.9.0-3) experimental; urgency=low
 .
   * Strict the compilation into Linux systems because of some kernel
     dependency (fo example tproxy)
 .
 The person named in the Author field signed this changelog entry.
Author: SZALAY Attila <sasa@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- zorp-3.9.0.orig/pylib/kznf/Makefile
+++ zorp-3.9.0/pylib/kznf/Makefile
@@ -99,7 +99,7 @@ CCDEPMODE = depmode=gcc3
 CFLAGS =  -O2  -Wall -W -Werror-implicit-function-declaration -Wno-pointer-sign -g -D_GNU_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
 CPP = gcc -E
 CPPFLAGS = -I/home/sasa/src/debian/zorp/zorp-3.9.0/zorp-3.9.0/lib -I/home/sasa/src/debian/zorp/zorp-3.9.0/zorp-3.9.0  -D_GNU_SOURCE -pthread -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include     -I/usr/include/python2.6  
-CURRDATE = Fri, 11 Mar 2011 22:30:25 +0100
+CURRDATE = Fri, 11 Mar 2011 22:30:57 +0100
 CYGPATH_W = echo
 DEFS = -DHAVE_CONFIG_H
 DEPDIR = .deps
--- zorp-3.9.0.orig/pylib/kznf/kznf/Makefile
+++ zorp-3.9.0/pylib/kznf/kznf/Makefile
@@ -59,7 +59,7 @@ CCDEPMODE = depmode=gcc3
 CFLAGS =  -O2  -Wall -W -Werror-implicit-function-declaration -Wno-pointer-sign -g -D_GNU_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
 CPP = gcc -E
 CPPFLAGS = -I/home/sasa/src/debian/zorp/zorp-3.9.0/zorp-3.9.0/lib -I/home/sasa/src/debian/zorp/zorp-3.9.0/zorp-3.9.0  -D_GNU_SOURCE -pthread -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include     -I/usr/include/python2.6  
-CURRDATE = Fri, 11 Mar 2011 22:30:25 +0100
+CURRDATE = Fri, 11 Mar 2011 22:30:57 +0100
 CYGPATH_W = echo
 DEFS = -DHAVE_CONFIG_H
 DEPDIR = .deps
